pkg/operator/controller.go

```diff
@@ -90,8 +90,6 @@ type wqKey struct {
 	kind      string // object kind
 	namespace string // object namespace
 	name      string // object name
-	//nolint:unused
-	event string // object event type (add/update/delete) or pass the full object on delete
```

Comment:
Unless there are backward compatibility constraints (e.g. public facing API), I prefer to just removed unused code/fields. I won't block this PR for this reason, though.